sphinx/builders/latex/theming.py

```diff
@@ -0,0 +1,118 @@
+"""
+    sphinx.builders.latex.theming
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Theming support for LaTeX builder.
+
+    :copyright: Copyright 2007-2019 by the Sphinx team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import configparser
+from os import path
+from typing import Dict
+
+from sphinx.application import Sphinx
+from sphinx.config import Config
+from sphinx.errors import ThemeError
+from sphinx.locale import __
+from sphinx.util import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Theme:
+    """A set of LaTeX configurations."""
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.docclass = name
+        self.wrapperclass = name
+        self.toplevel_sectioning = 'chapter'
```

Comment:
I think there are some candidates for theme's configurations:
* papersize
* pointsize
* fncychap
* classoptions (ex. openany, twocolumn)

And users can overrided these theme's settings via `latex_elements` if they want.